dvc/commands/add.py

```diff
@@ -127,6 +128,16 @@ def add_parser(subparsers, parent_parser):
         help="Don't recreate links from cache to workspace.",
     )
     parser.set_defaults(relink=True)
+    parser.add_argument(
+        # Do we want a short code here?
```

Comment:
I think it's fine if an option like this is long since we want people to be very explicit in choosing this option.